Controllers/ExampleController.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Bolt\Extension\YourName\ExtensionName\Controllers;
+
+use Bolt\Application;
+use Symfony\Component\HttpFoundation\JsonResponse;
+
+class ExampleController
+{
+    /**
+     * Bolt Application instance
+     *
+     * @var Application
+     */
+    private $app;
+
+    /**
+     * Extension configuration
+     *
+     * @var array
+     */
+    private $config;
+
+    function __construct(Application $app, $config)
```

Comment:
It is required for PSR-2 compliance:
* 4.2 [Visibility MUST be declared on all properties.](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#42-properties)
* 4.3 [Visibility MUST be declared on all methods.](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md#43-methods)